### DISCUSSION TITLE
GitHub Achievements

### DISCUSSION THREAD

**Discussion by bluetch** [community/community#36082] [Other Feature Feedback, Questions, & Ideas] [answered]
I have just closed an issue from another user in 5 minutes, however, the badge didn't show up.
How can I make sure achievement reaches or not? is there updating delay time?

**Comment by jooy2** [comment, marked as answer]
Merge open PRs without separate reviews or comments to earn badges.
Or even get a badge if your PR is merged by someone.

Badge renewal may take some time.

**Comment by defuj** [comment]
Just wait 👍🏻

**Comment by ghost** [comment]
Badge renewal may take some time.
